doc/elements-confidential-transactions.md

```diff
@@ -0,0 +1,480 @@
+the `importblindingkey` RPC.
+
+`importblindingkey`
+Imports the private blinding key associated with an address.
+
+`importissuanceblindingkey`
+Imports a private blinding key that may be used to unblind the amounts
+on an issuance input, or reissue additional amounts of an asset (using
+reissuance tokens). 
+
+`importmasterblindingkey`
+***Use with caution!*** Importing a master blinding key into a wallet will
+erase the existing master blinding key, potentially making all
+addresses owned by that wallet unspendable.
+
+
+
+### Wallet - Transactions
+
+`blindrawtransaction`
+Blinds the outputs of a raw transaction (as might be created by `createrawtransaction`)
+
+
+`unblindrawtransaction`
+Unblinds a raw transaction, using addresses and blinding keys known to
+the wallet (they must have been previously imported, see "Third-party
+Unblinding" below.
+
+
+
+## Third-party Unblinding (Watch-only wallets)
+
+A third-party may be granted the ability to unblind the amounts and
+assets in a confidential transaction, without being able to spend the
+transaction’s UTXOs. Using Elements, the third-party would create a
+"watch-only wallet" for the addresses in question, and import the
+private blinding keys for those addresses.  
+
+Let’s suppose that Alice has sent a confidential transaction to
+Bob. Bob wants Victor to be able to see what and how much was sent. 
+
+Victor, with Bob’s help, creates a watch-only wallet in Elements:
+
+1. for each address of interest A, Victor runs the RPC `importaddress
+<A>`
+
+2. Bob exports the blinding key for address A using `dumpblindingkey
+<A>` and shares it with Victor.
+
+3. Victor imports the blinding key for the address using `importblindingkey <A> <blinding key>`
+
+Once the blinding key is imported, the Elements wallet will treat the
+Confidential address address as watch-only, and its outputs will be
+visible in transaction details and in the wallet balance.  
+
+Please note that if Bob reuses an address A, Victor will also be able
+to see the amounts and values in any transaction sending to A. 
+
+Alternatively, a watch-only wallet may import the master blinding key
+of another wallet. The watch-only wallet would then be able to view
+the UTXOs for any confidential address created by the original
+wallet. 
+
+Elements is not required to unblind a confidential transaction. Given
+the confidential address’ blinding key, the rangeproof for the output
+can be rewound, revealing the blinding factors and actual amounts and
+assets that were committed to.  
+
+Please see the [Elements Project
+tutorial](https://elementsproject.org/elements-code-tutorial/advanced-examples)
+for examples of how to unblind with Elements.
+
+
+## Wallet Considerations
+
+An Elements wallet has a "master blinding key", from which all
+blinding keys for that wallet are deterministically derived. A
+blinding key for an address is generated as HMAC_SHA256(master
+blinding key, <address ScriptPubKey>). See SLIP-0077 [6]. 
```

Comment:
This is not rendered correctly (see the corresponding text in the rendered markdown document).

I believe you have to escape the angle brackets.

Alternatively, perhaps this equation can go inside a code block, like this:

```suggestion
blinding key for an address is generated as `HMAC_SHA256(master
blinding key, <address ScriptPubKey>)`. See SLIP-0077 [6]. 
```